src/app/features/characters/components/find-character-form/find-character-form.component.ts

```diff
@@ -0,0 +1,72 @@
+import { Component, OnDestroy } from '@angular/core';
+import { FormControl, FormGroup, ReactiveFormsModule } from '@angular/forms';
+import { Router } from '@angular/router';
+import { MarvelChar } from '@characters/models/characters.model';
+import { CharactersMarvelService } from '@characters/services/characters-marvel.service';
+import { Subscription } from 'rxjs';
+
+enum EMSG {
+  clear = '',
+  emptyInput = 'empty input',
+  notFound = 'not found',
+  found = 'found'
+}
+
+@Component({
+  selector: 'find-character-form',
+  standalone: true,
+  imports: [ ReactiveFormsModule ],
+  templateUrl: './find-character-form.component.html',
+  styleUrl: './find-character-form.component.scss'
+})
+export class FindCharacterFormComponent implements OnDestroy {
+  static EMSG = EMSG;
+
+  get enumMessages() {
```

Comment:
the name must tell what exactly messages are they

Reply:
changed